python/lsst/pipe/analysis/visitAnalysis.py

```diff
@@ -415,21 +415,6 @@ def runDataRef(self, dataRefList, tract=None, subdir=""):
                     if "sky_source" in schema:
                         baseGoodSky = catalog["sky_source"]
                         skySrcCat = catalog[baseGoodSky].copy(deep=True)
-                        if "deblend_scarletFlux" in schema:
```

Comment:
Here is where things are inconsistent, because this module does not support backwards compatibility, as skipping sky objects was added later.